### ISSUE TITLE
UNWIND opcode in unwind.c bug

### ISSUE THREAD

**Issue by masinter** [Interlisp/medley#101]
I suspect this opcode is rare -- the same code compiled with the Interlisp compiler and common lisp compiler are different, and UNWIND only appears in the CL-compiled code.

Looking at unwind.c doesn't explain how it gets by without any byteswapping.

I could BREAK(\UNWIND.UFN) and not hit a breakpoint. The UFN gets called on a Pi and not on an x86.
so the problem is apparent right at the beginning
```
  if (endptr > cstkptr) {
    CurrentStackPTR = (DLword *)cstkptr;
    ERROR_EXIT(tos);
  }
```

**Comment by masinter** [comment]
#54 and 
#19  might be related

**Comment by nbriggs** [comment]
The same problem happens on a BeagleBone Black, an ARMv7l processor running Debian 9 or 10 as well.
By the time the UFN fails (executing a PUTBASE.N opcode) the data on the stack seems wrong.

**Comment by nbriggs** [comment]
OK -- got this one.  The root cause is that the code is not prepared to deal with the Lisp stack being allocated in host memory that is above 0x7FFFFFFF, and through sloppy coding mistakes normal returns for error returns.

**Comment by nbriggs** [comment]
Fixed by PR Interlisp/maiko#158
